deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/Utils.java

```diff
@@ -24,4 +27,21 @@ static boolean hasEmptyOrNullString(String[] strings) {
     }
     return false;
   }
+
+  /** Strips off {@link #PARAM} and {@link #RETURN}, and removes line separator. */
+  static String formatJavaDoc(String str) {
```

Comment:
Would go well in the DocGenerator class if you go with my separate class suggestion